contracts/payloads/IGP43/PayloadIGP43.sol

```diff
@@ -0,0 +1,692 @@
+
+    FluidVaultFactory public constant VAULT_FACTORY = 0x324c5Dc1fC42c7a4D43d92df1eBA58a54d13Bf2d;
+
+    FluidVaultResolver public constant FluidVaultT2Resolver; // TODO add address here
+
+    FluidVaultResolver public constant FluidVaultT3Resolver; // TODO add address here
+
+    FluidVaultResolver public constant FluidVaultT4Resolver; // TODO add address here
+
+    uint256 public constant wstETH_ETH_Dex; // TODO add address here
+    uint256 public constant USDC_USDT_Dex; // TODO add address here
+    uint256 public constant cbBTC_WBTC_Dex; // TODO add address here
+
+    address public constant wstETH_ETH_Smart; // TODO add address here
+    address public constant ETH_USDC_USDT; // TODO add address here
+    address public constant wstETH_USDC_USDT; // TODO add address here
+    address public constant weETH_USDC_USDT; // TODO add address here
+    address public constant WBTC_USDC_USDT; // TODO add address here
+    address public constant cbBTC_USDC_USDT; // TODO add address here
+    address public constant sUSDe_USDC_USDT; // TODO add address here
+    address public constant cbBTC_WBTC_Smart; // TODO add address here
+    address public constant cbBTC_WBTC_USDC; // TODO add address here
+    address public constant cbBTC_WBTC_USDT; // TODO add address here
+
+    constructor() {
+        ADDRESS_THIS = address(this);
+    }
+
+    function propose(string memory description) external {
+        require(msg.sender == PROPOSER || msg.sender == TEAM_MULTISIG, "msg.sender-not-proposer-or-multisig");
+
+        uint256 totalActions = 1;
+        address[] memory targets = new address[](totalActions);
+        uint256[] memory values = new uint256[](totalActions);
+        string[] memory signatures = new string[](totalActions);
+        bytes[] memory calldatas = new bytes[](totalActions);
+
+        // Action 1: call executePayload on timelock contract to execute payload related to Fluid
+        targets[0] = address(TIMELOCK);
+        values[0] = 0;
+        signatures[0] = "executePayload(address,string,bytes)";
+        calldatas[0] = abi.encode(ADDRESS_THIS, "execute()", abi.encode());
+
+        uint256 proposedId = GOVERNOR.propose(targets, values, signatures, calldatas, description);
+
+        require(proposedId == PROPOSAL_ID, "PROPOSAL_IS_NOT_SAME");
+    }
+
+    function execute() external {
+        require(address(this) == address(TIMELOCK), "not-valid-caller");
+
+        // Action 1: Give allowance on Liquidity Layer to initial 2 DEXes
+        action1();
+
+        // Action 2: Give allowance on Liquidity Layer to new vaults
+        action2();
+    }
+
+    function verifyProposal() external view {}
+
+    /**
+     * |
+     * |     Proposal Payload Actions      |
+     * |__________________________________
+     */
+
+    /// @notice Action 1: Setting supply and borrow limit for Dexes
+    function action1() internal {
+        Pool memory pool1_ = DEX_RESERVES_RESOLVER.getPool(wstETH_ETH_Dex);
```

Comment:
@0xprinc don't use resolver as of now. fetch the details directly from constantViews